crypto/asn1/i2d_evp.c

```diff
@@ -0,0 +1,147 @@
+/*
+ * Copyright 1995-2020 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License 2.0 (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+/* We need to use some deprecated APIs to support the legacy bits */
+#define OPENSSL_SUPPRESS_DEPRECATED
```

Comment:
What about keeping the public key functions separate as I suppose these are the only that need the deprecation suppressed?

Reply:
So, more internal symbols in the static library?  I get the idea, but there's so little in this file that isn't deprecated / legacy stuff that I wonder if that's worth it...

Reply:
OK, it's fine by me as is.